src/swarmus_ros_simulation/launch/slam_simulation.launch

```diff
@@ -5,10 +5,9 @@
     <arg name="robot_name" default="pioneer_0"/>
```

Comment:
Shouldn't we name the file with prefix pioneer_0 like the other files as it refers to this robot only right now?

Reply:
Except some RVIZ launchfile, I don't see other files with the pioneer_0 prefix. However, since we use a lot the pioneer_0 as the default robot for control, for visualization, for single robot simulation and for our real robots' name, it could be specified in the readme